TransformationSystem/Client/test/Test_Client_TransformationSystem.py

```diff
@@ -12,6 +12,21 @@
 
 #############################################################################
 
+
+class reqValFake_C(object):
```

Comment:
Ah True ! However, the logic inside the RequestValidator does not need the CS, so just making it happy at initialization will allow it to do its real work then. Maybe you can just mock the init rather than the whole object ?